db/migrate/20161018001343_create_tutorials.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTutorials < ActiveRecord::Migration
+  def change
+    create_table :tutorials do |t|
+      t.string :name
```

Comment:
One thing to think about on your table structures will be which data must *always* be present and which can be sometimes present. You can enforce the always ones at the database level with `null: false`. This makes it literally impossible to put a null value in postgres